### DISCUSSION TITLE
[PSD7] Exercise 2.27

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#155] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210018761-bfe9bb2a-ed86-4349-a75e-445fc89eb979.png)

**Comment by Elia-Belli** [comment]
![BP2](https://user-images.githubusercontent.com/107944476/211665302-f2dd0d3f-5081-430f-850b-c21c3eb76e7b.jpg)
$Y = (ABC + \bar D)+(\bar E (\bar F + \bar G))$

**Comment by fraxb04** [comment]
corretto
